src/views/CreateProposal/index.tsx

```diff
@@ -327,16 +380,16 @@ function CreateProposal() {
                     marginLeft: "16px",
                   }}
                 >
-                  Memo
+                  The duration after which the funding period will end
```

Comment:
Please change the duration text to 'Number of days to run the crowdfunding campaign'